index.html

```diff
@@ -1,11 +1,129 @@
-  <title>Document</title>
+  <link rel="stylesheet" href="./css/styles.css" />
+    <link href="https://fonts.googleapis.com/css2?family=Raleway:ital,wght@0,700;1,700&family=Roboto:ital,wght@0,400;0,500;0,700;0,900;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+    <title>Homework 2</title>
 </head>
+
 <body>
+  <header class="header">
+    <div class="container">
+      <a href="index.html" class="logo">WebStudio</a>
+
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a class="nav-link" href="index.html">Студія</a>
+          </li>
+          <li>
+            <a class="nav-link" href="portfolio.html">Портфоліо</a>
+          </li>
+          <li>
+            <a class="nav-link" href="#contacts">Контакти</a>
+          </li>
+        </ul>
+      </nav>
   
+      <address class="contact">
+        <a class="contact-link" href="mailto:info@devstudio.com">info@devstudio.com</a>
+        <a class="contact-link" href="tel:+380961111111">+38 096 111 11 11</a>
+      </address>
+    </div>
+  </header>
+
+  <main>
+    <section class="hero">
+      <h1 class="hero-title">
+        ЕФЕКТИВНІ РІШЕННЯ ДЛЯ ВАШОГО БІЗНЕСУ
+      </h1>
+
+      <button type="button" class="hero-button">
+        Замовити послугу
+      </button>
+    </section>
+
+    <section class="features">
+      <div class="container">
+        <ul class="features-list">
+          <li class="features-item feature">
+            <h3 class="feature-title">Увага до деталей</h3>
+            <p class="feature-text">Ідейні міркування, і навіть початок повсякденної роботи з формування позиції.</p>
+          </li>
+          <li class="features-item feature">
+            <h3 class="feature-title">Пунктуальність</h3>
+            <p class="feature-text"> Завдання організації, особливо рамки і місце навчання кадрів тягне у себе.</p>
+          </li>
+          <li class="features-item feature">
+            <h3 class="feature-title">Планування</h3>
+            <p class="feature-text">Так само консультація з широким активом значною мірою зумовлює.</p>
+          </li>
+          <li class="features-item feature">
+            <h3 class="feature-title">Сучасні технології</h3>
+            <p class="feature-text">Значимість цих проблем настільки очевидна, що реалізація планових завдань.</p>
+          </li>
+        </ul>
+      </div>
+    </section>
+
+    <section class="services">
+      <!-- container-->
+      <h2>Чим ми займаємось</h2>
+
+      <ul class="activities">
+        <li>
+          <img class="activities-item" src="./images/plan.jpg" width="370" height="294" alt="plan">
+        </li>
+        <li>
+          <img class="activities-item" src="./images/phone.jpg" width="370" height="294" alt="size">
+        </li>
+        <li>
+          <img class="activities-item" src="./images/colors.jpg" width="370" height="294" alt="design">
+        </li>
+      <!-- container-->
+
```

Comment:
section content must be wrapped in the `div.container` element

Reply:
make sure to add classes to all elements

Reply:
Div.container is homework 3
H2 class="activities-head" . Is it good?
li class=" activities-item". Is it good name's class?

Reply:
better use `services` for section
section title `services-title`
list `services-list`
item `services-item`